apps/segfault-on-filtered-vector-search/run.py

```diff
@@ -0,0 +1,133 @@
+import weaviate
+import datetime
+import time
+from loguru import logger
+from typing import Optional
+import random
+import numpy as np
+import argparse
+import string
+
+def reset_schema(client: weaviate.Client):
+    client.schema.delete_all()
+    class_obj = {
+        "vectorizer": "none",
+        "vectorIndexConfig":{
+            "efConstruction": 32,
+            "maxConnections": 4,
+        },
+        "class": "Example",
+        "invertedIndexConfig":{
+            "indexTimestamps":False,
+        },
+        "properties": [
+            {
+                "dataType": [ "boolean" ],
+                "name": "bool_field",
+            },
+            {
+                "dataType": [ "string" ],
+                "name": "string_field",
+            },
+        ]
+    }
+
+    client.schema.create_class(class_obj)
+
+# https://xkcd.com/221/
+random_words = ['ZUxO', 'wpzF', 'Dexz', 'sfaE', 'nTxi', 'Dtkw', 'fgiO', 'FnFV', 'FNkW', 'inkI', 'zunb', 'AaQg', 'CdzQ', 'PIaQ', 'iPlx', 'OzKq', 'Yvbn', 'QqeX', 'lFGF', 'cTRL']
+
+def handle_errors(results: Optional[dict]) -> None:
+    """
+    Handle error message from batch requests logs the message as an info message.
+    Parameters
+    ----------
+    results : Optional[dict]
+        The returned results for Batch creation.
+    """
+
+    if results is not None:
+        for result in results:
+            if (
+                'result' in result
+                and 'errors' in result['result']
+                and 'error' in result['result']['errors']
+            ):
+                for message in result['result']['errors']['error']:
+                    logger.error(message['message'])
+
+def load_records(client: weaviate.Client, max_records=100000):
+    global random_words
```

Comment:
pretty sure you do not need to do that as you've defined it at the top level

Reply:
You're probably right, I'll try removing it and see if all is still green.